controllers/backup/cronjob.go

```diff
@@ -114,6 +115,41 @@ func buildJobTemplate(conf *config.Config, dbcr *kciv1alpha1.Database) (batchv1b
 	}, nil
 }
 
+func getResourceRequirements(conf *config.Config) v1.ResourceRequirements {
+	resourceRequirements := v1.ResourceRequirements{}
+
+	requests := make(v1.ResourceList)
+	cpuRequests, err := resource.ParseQuantity(conf.Backup.Resource.Requests.Cpu)
```

Comment:
if you not handle any error case, you could ignore err at all ... related if an error happen, an empty Quantity type is return. https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L275

so you could maybe simplify this by

```go
resourceRequirements := v1.ResourceRequirements{}

resourceRequirements.Requests = v1.ResourceList{
    cpu:    resource.ParseQuantity(conf.Backup.Resource.Requests.Cpu),
    memory: resource.ParseQuantity(conf.Backup.Resource.Requests.Memory)
}
```

but maybe I'm also wrong 😅

Reply:
somehow empty string is set value 0